skbio/io/format/genbank.py

```diff
@@ -0,0 +1,683 @@
+"""
+Genbank format (:mod:`skbio.io.format.genbank`)
+===============================================
+
+.. currentmodule:: skbio.io.format.genbank
+
+Genbank format (Genbank Flat File Format) stores sequence and its annotation
+together. The start of the annotation section is marked by a line beginning
+with the word "LOCUS". The start of sequence section is marked by a line
+beginning with the word "ORIGIN" and the end of the section is marked by a line
+with only "//".
+
+The Genbank file usually ends with .gb or sometimes .gbk. The Genbank format
+for protein has been renamed to Genpept. The Genbank (for nucleotide) and
+Genpept are essentially the same format.
+
+An example of a Genbank file can be see here:
+<http://www.ncbi.nlm.nih.gov/Sitemap/samplerecord.html>
+
+Format Support
+--------------
```

Comment:
We don't have a great way of indicating [API stability](http://scikit-bio.org/docs/latest/user/api_stability.html) for file formats yet. For now, can you please add the following line?

```
**State: Experimental as of 0.4.0-dev.**
```